extensionsGallery.json

```diff
@@ -3765,6 +3765,63 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "77",
```

Comment:
please bump the resultCount at the end of the file by one since you are adding a new extension. also please update the extensionsGallery-insider.json file so that your extension is available for ADS insider builds as well.